tests/Node.py

```diff
@@ -1375,10 +1377,13 @@ def verifyAlive(self, silent=False):
             # mark node as killed
             self.pid=None
             self.killed=True
+            if logStatus: Utils.Print("Determined node(formerly pid=%s) is killed" % (pid))
             return False
         except PermissionError as ex:
+            if logStatus: Utils.Print("Determined node(formerly pid=%s) is killed" % (pid))
             return True
 
+        if logStatus: Utils.Print("Determined node(pid=%s) is alive (kill command executed)" % (self.pid))
```

Comment:
This didn't call `kill`, well technically it did, but the kill is kill 0 which is just check the status. I think you should remove "(kill command executed)"